account_credit_control/migrations/9.0.1.0.0/post-migration.py

```diff
@@ -40,7 +41,37 @@ def migrate_followup_data(env):
             })
 
 
+def set_followup_data(env):
+    today = fields.Date.context_today
+    policy = env['credit.control.policy.level'].search([])[:-1]
+    if policy:
+        env.cr.execute("""
+            SELECT aml.* FROM account_move_line aml, account_account aa
+            WHERE aml.account_id = aa.id
+            AND invoice_id IS NOT NULL
+            AND aa.internal_type = 'receivable'
+            AND aml.reconciled IS NULL
+            AND aml.date_maturity < %s
+        """, today)
+        data = env.cr.fetchall()
+        for line in data:
+            env['credit.control.line'].create({
+                'date': today,
+                'date_due': line['date_maturity'],
```

Comment:
For having this working, you have to make `dictfetchall`, no `fecthall`.

Reply:
😄 Sure!